Big_Integer/main.cpp

```diff
@@ -0,0 +1,476 @@
+#include<bits/stdc++.h>
+
+
+using namespace std;
+
+
+class Integer {
+    int sign;
+    string value;
+
+public:
+    Integer() {
+        //Default Empty Constructor::Dont use delegation...
+    }
+    //positive - negative value separator
+    Integer(string passed) {
+        if (passed[0] == '0') {
+            sign = 0;
+            value = "0";
+        } else if (passed[0] == '-'){
+            sign = -1;
+            value = passed.substr(1); // Copy substring after pos 1
+        } else {
+            sign = +1;
+            value = passed;//if positive store as it's
+        }
+    }
+    Integer(string passed, int Sign) {
+        value = passed;
+        sign = Sign;
+    }
+
+    //Day 4: if  integer range numbers  are passed then  only will work fine.....preferably pass string values..
+    Integer(int passed) {    //Remember Remove this as we will be passing string as a parameter
```

Comment:
Codacy found an issue: [Class 'Integer' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6290669)